index.html

```diff
@@ -48,78 +57,127 @@ <h2>Tonic</h2>
                 </ul>
+                <p class="mobileP">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="deskP">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                <div class="mobileTeck">
+                    <ul class="teck-list">
+                        <li><button class="mobile-btn" type="submit">HTML</button></li>
+                        <li><button class="mobile-btn" type="submit">CSS</button></li>
+                        <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <div class="deskTeck">
+                    <ul class="teck-list">
+                        <li><button class="mobile-btn" type="submit">HTML</button></li>
+                        <li><button class="mobile-btn" type="submit">Ruby on Rails</button></li>
+                        <li><button class="mobile-btn" type="submit">CSS</button></li>
+                        <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <button class="see-mobile-btn" type="submit">See Projects</button>
             </div>
-            <button class="see-mobile-btn" type="submit">See Projects</button>
         </section>
     </section>
     <!-- about area -->
     <section class="about">
-        <h5>About Myself</h5>
-        <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-        <span>LET'S CONNECT</span>
-        <ul class="media-links">
-            <li><a href="#"><i class="fa fa-twitter"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li> 
-        </ul>
-        <button class="resume-mobile-btn" type="submit">Get my resume</button>
+        <div class="about-grid-column">
+            <h5>About Myself</h5>
+            <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <span class="connect">LET'S CONNECT</span>
+            <ul class="media-links" id="connect-media-links">
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
```

Comment:
- [ ] As mentioned before please make sure that the font-weight is not bold for the `About us description`
 
<img width="573" alt="Screenshot 2023-07-15 at 7 09 17 PM" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/44798044/13424519-52d4-45ad-ae36-38aac07ac328">